doc/docs/lexerdevelopment.rst

```diff
@@ -164,16 +164,16 @@ To test the new lexer, store an example file with the proper extension in
 ``tests/examplefiles``.  For example, to test your ``DiffLexer``, add a
 ``tests/examplefiles/example.diff`` containing a sample diff output.
 
-Now you can use pygmentize to render your example to HTML:
+Now you can use ``python -m pygments`` to render your example to HTML:
 
 .. code-block:: console
 
-    $ ./pygmentize -O full -f html -o /tmp/example.html tests/examplefiles/example.diff
+    $ python -m pygments -O full -f html -o /tmp/example.html tests/examplefiles/example.diff
 
-Note that this explicitly calls the ``pygmentize`` in the current directory
-by preceding it with ``./``. This ensures your modifications are used.
-Otherwise a possibly already installed, unmodified version without your new
-lexer would have been called from the system search path (``$PATH``).
+Note that this explicitly calls the ``pygments`` module in the current
+directory. This ensures your modifications are used. Otherwise a possibly
```

Comment:
Now that this isn't bound to ``./pygmentize`` existing, it's better to state that this must be run from the root of the checkout.  The "Otherwise..." is still useful in this case.

Reply:
Improved!